test/unit/org/apache/cassandra/cql3/validation/operations/InsertUpdateIfConditionTest.java

```diff
@@ -459,7 +459,7 @@ public void testDropCreateTableIfNotExists() throws Throwable
     @Test
     public void testDropCreateIndexIfNotExists()
     {
-        String tableName = createTable("CREATE TABLE %s (id text PRIMARY KEY, value1 blob, value2 blob)with comment = 'foo'");
+        String tableName = createTable("CREATE TABLE %s (id text PRIMARY KEY, value1 text, value2 blob)with comment = 'foo'");
```

Comment:
Why this change?

Reply:
SAI does not support indexes on blobs. This tests does not need a blob, text (which SAI supports) suffices.